web/gds-user-ui/src/components/CertificateReview/index.tsx

```diff
@@ -89,6 +89,13 @@ const CertificateReview = () => {
     }
   }, [isTestNetSent, isMainNetSent, dispatch]);
 
+  const handleJumpToLastStep = (e: React.FormEvent) => {
+    e.preventDefault();
+    jumpToLastStep();
+    setIsTestNetSent(false);
```

Comment:
Setting the state back to false here ensures that the confirmation modal will not appear once the user navigates away from the submit page, unless they submit a registration for another network.